examples/todo/src/Component/Task.purs

```diff
@@ -1,71 +1,82 @@
-module Component.Task where
+module Example.Todo.Component.Task where
 
 import Prelude
 
-import Control.Monad.State as CMS
 import Data.Maybe (Maybe(..))
+import Example.Todo.Model (Task, initialTask)
 import Halogen as H
 import Halogen.HTML as HH
 import Halogen.HTML.Events as HE
 import Halogen.HTML.Properties as HP
-import Model (Task)
 
```

Comment:
I used this todo example as my gateway to learning Halogen v5. The one thing I'm feeling uncertain on is how to deal with the situation where you want a query AND action for the same thing. In this case, that is "SetCompleted". I just duplicated the command and appended a ' in this case. This may or may not be idiomatic, I have no idea. It's not perfectly DRY, but neither is it terrible repetition either. Granted, it might not be terrible only because it's a tiny example.